fastapi_injector/attach.py

```diff
@@ -1,12 +1,23 @@
+from typing import Optional
+
 from fastapi import FastAPI
-from injector import Injector
+from injector import Injector, InstanceProvider, singleton
 
 from fastapi_injector.exceptions import InjectorNotAttached
+from fastapi_injector.request_scope import RequestScopeOptions
 
 
-def attach_injector(app: FastAPI, injector: Injector) -> None:
-    """Call this function on app startup to attach an injector to the app."""
+def attach_injector(
+    app: FastAPI, injector: Injector, options: Optional[RequestScopeOptions] = None
+) -> None:
```

Comment:
If I'm not mistaken, this could be reduced to 
```python
def attach_injector(
    app: FastAPI, injector: Injector, options: RequestScopeOptions = RequestScopeOptions()
) -> None:
```
Then it would simply use the defaults from the `RequestScopeOptions` class. This never gets mutated, so creating an instance as a default argument should be safe.

Reply:
Updated as requested. Also, there can't be many scenarios where `attach()` is called multiple times!

Reply:
BTW I didn't say earlier, but thanks for your work on creating and maintaining this library, I really appreciate it 👍🏻